Bundle/WidgetBundle/Entity/Traits/LinkTrait.php

```diff
@@ -12,75 +12,84 @@
  */
 trait LinkTrait
 {
+    /**
+     * @ORM\OneToOne(targetEntity="Victoire\Bundle\CoreBundle\Entity\Link", cascade={"persist", "remove"})
+     * @ORM\JoinColumn(name="link_id", referencedColumnName="id", nullable=true, onDelete="SET NULL")
+     **/
+    protected $link;
 
     /**
      * @var string
      *
      * @ORM\Column(name="url", type="string", length=255, nullable=true)
+     * @deprecated please run the victoire:legacy:linkMigrator command
      */
-    protected $url;
+    public $url;
```

Comment:
I'm affrait of an incompatibility with Doctrine Proxies here. The generated proxies could generate getter/setters for these properties and so maybe override ours.
See that: http://stackoverflow.com/questions/4090609/how-can-public-fields-break-lazy-loading-in-doctrine-2